cpp/arcticdb/version/version_map_entry.hpp

```diff
@@ -35,6 +33,12 @@ inline constexpr bool is_partial_load_type(LoadType load_type) {
     return load_type == LoadType::LOAD_DOWNTO || load_type == LoadType::LOAD_FROM_TIME;
 }
 
+// Used to specify whether we want to load all or only undeleted versions
+enum class ToLoad : uint32_t {
```

Comment:
Slightly odd name. Maybe something like LoadObjective or LoadRequirement?

Reply:
Thanks! `LoadObjective` is way better.